app1/admin.py

```diff
@@ -1,3 +1,19 @@
-from django.contrib import admin
+#from django.contrib import admin
 
 # Register your models here.
+from django.contrib import admin
+from .models import YourModel  # Import your model(s) here
+
+@admin.register(YourModel)
```

Comment:
**Suggestion**: Ensure that the 'YourModel' class is correctly imported and registered for admin operations. <br> **Code Suggestion**: 
 ```
 +@admin.register(YourModel)
 ```